tests/bluetooth/bsim_bt/bsim_test_mesh/_mesh_test.sh

```diff
@@ -52,7 +54,6 @@ function RunTest(){
   for process_id in $process_ids; do
     wait $process_id || let "exit_code=$?"
   done
-  exit $exit_code #the last exit code != 0
```

Comment:
The shell script must fail when any device return code is !=0. (otherwise failures won't be reported in CI)
Instead of trying to chain several tests in shell script (like with `tests/bluetooth/bsim_bt/bsim_test_mesh/tests_scripts/transport/unicast.sh`)
maybe add a `unicast_lowlat.sh` and a `msg_mesh_low_lat.sh`

(Apart from the "the test must fail" issue, note that `process_ids` and `exit_code`, (and the possible `SKIP` and `EXTRA_DEVS`) are global for the calling script, so calling Runtest several times is a bad idea)